opsdroid/connector/matrix/connector.py

```diff
@@ -264,6 +277,8 @@ async def _parse_sync_response(self, response):
                     if event.sender != self.mxid:
                         if event.source["type"] == "m.room.member":
                             event.source["content"] = event.content
+                        if isinstance(event, nio.MegolmEvent):
+                            _LOGGER.error(f"Failed to decrypt event {event}")
```

Comment:
It seems that this is the only thing that is making cov complain. Can you write a silly test to get that log line covered?

Reply:
ffs codecov

Reply:
```suggestion
                            _LOGGER.error(f"Failed to decrypt event {event}")  # pragma: no cover
```